java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -283,6 +283,7 @@ protected RecordReaderImpl(ReaderImpl fileReader,
     Consumer<OrcFilterContext> filterCallBack = null;
     BatchFilter filter = FilterFactory.createBatchFilter(options,
                                                          evolution.getReaderBaseSchema(),
+                                                         evolution.isSchemaEvolutionCaseAware,
```

Comment:
For consistency, could you use the getter function?

Reply:
Good point, changed.